src/instances/rundler.ts

```diff
@@ -304,9 +324,21 @@ export type RundlerParameters = {
     /**
      * Choice of what sender type to use for transaction submission.
      * @default raw
-     * options: raw, conditional, flashbots, polygon_bloxroute
+     * options: raw, flashbots, polygon_bloxroute
+     */
+    sender?: 'raw' | 'flashbots' | 'polygonBloxroute'
+
+    /**
+     * Use the submit URL for transaction status checks.
+     * @default false
+     */
+    use_submit_for_status?: boolean
```

Comment:
```suggestion
    useSubmitForStatus?: boolean
```

These should automatically be converted to snake_case at runtime.